ui/static_table/viewmodels/OBATableRow.m

```diff
@@ -23,7 +23,7 @@ + (void)load {
 }
 
 + (instancetype)tableRowWithTitle:(NSString*)title action:(void (^)())action {
-    return [[self alloc] initWithTitle:title action:action];
+    return [[OBATableRow alloc] initWithTitle:title action:action];
```

Comment:
A fair point. The issue this solves is disambiguating the `copy` selector from a couple of overrides in a pod project (a warning). I'll look for a construct that identifies the type of self automatically.